Xamarin.Forms.Platform.Android/Platform.cs

```diff
@@ -427,6 +427,10 @@ internal static void SetPageContext(BindableObject bindable, Context context)
 
 		internal void UpdateActionBar()
 		{
+			if (ActionBar == null) //Fullscreen theme doesn't have action bar
```

Comment:
`ActionBar` is directly referenced in dozens of other places in this file without a `null` check. Does it not error anywhere else?